src/ponyfills.ts

```diff
@@ -1,6 +1,6 @@
 import { isBrowser } from './browser'
 
-const elementProto = isBrowser ? Element.prototype : {}
+const elementProto = isBrowser ? Element.prototype : ({} as any)
```

Comment:
```suggestion
const elementProto: Record<string, any> = isBrowser ? Element.prototype : {}
```

`Record<TKey, TValue>` evaluates to `{ [key: TKey]: TValue }` and is common to describe an actual object (not non-primitive, like `object`).